src/gps/dateTime.c

```diff
@@ -133,3 +133,118 @@ tiny_millis_t getUptime() {
 int getUptimeAsInt() {
    return (int) getUptime();
 }
+
+void getDateTimeFromEpochMillis(DateTime *dateTime, millis_t millis) {
```

Comment:
Thanks all for the feedback. I feel less dumb after this conversation :) By and large we pass things by reference in the codebase, except for maybe a few areas where we send tiny structs around (GPSPoint, maybe).  I think for the time being we can stay with this convention.

Going to close this PR and move on. Much thanks and respect for your opinions and wisdom.